src/Uno.UWP/Storage/StorageFolder.cs

```diff
@@ -176,42 +176,45 @@ internal StorageFolder(string name, string path)
 
 		public IAsyncOperation<StorageFile> CreateFileAsync(string desiredName) => CreateFileAsync(desiredName, CreationCollisionOption.FailIfExists);
 
-        public IAsyncOperation<StorageFile> CreateFileAsync(string path, CreationCollisionOption option) =>
+        public IAsyncOperation<StorageFile> CreateFileAsync(string desiredName, CreationCollisionOption option) =>
```

Comment:
This is a breaking change. Is it how the parameter is named under UWP ?

Reply:
desiredName